.github/workflows/tests.yml

```
@@ -0,0 +1,155 @@
+          cacheVersion: ${{inputs.cacheVersion}}
+          pnpmVersion: ${{inputs.pnpmVersion}}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.natToken }}
+          NODE_OPTIONS: ${{ inputs.nodeOptions }}
+
+      - uses: actions/cache@v3
+        id: build-cache
+        name: Restore build cache
+        with:
+          path: ./*
+          key: ${{ github.sha }}
+
+      - name: Unit tests server
+        run: yarn test:unit:server --ci
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.natToken }}
+          NODE_OPTIONS: ${{ inputs.nodeOptions }}
+
+  test-unit-client:
+    runs-on: self-hosted
+
+    steps:
+      - uses: actions/checkout@v3
+        name: Checkout
+
+      - uses: AstrumU/shared-workflows/setup@feature/KAN-206
+        name: setup env
+        with:
+          nodeVersion: ${{inputs.nodeVersion}}
+          packageManager: ${{inputs.packageManager}}
+          cacheVersion: ${{inputs.cacheVersion}}
+          pnpmVersion: ${{inputs.pnpmVersion}}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.natToken }}
+          NODE_OPTIONS: ${{ inputs.nodeOptions }}
+
+      - uses: actions/cache@v3
+        id: build-cache
+        name: Restore build cache
+        with:
+          path: ./*
+          key: ${{ github.sha }}
+
+      - name: Unit tests client
+        run: yarn test:unit:client --ci
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.natToken }}
+          NODE_OPTIONS: ${{ inputs.nodeOptions }}
+
+  test-int-client:
+    runs-on: self-hosted
+    strategy:
+      fail-fast: false
+      matrix:
+        client:
+          - skillset
+          - toolkit-mba
+
+    name: test-int-client-${{ matrix.client }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup kernel for node native, increase watchers
+        run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
+
+      - name: Install system packs
+        run: |
+          sudo apt update
+          sudo apt install -y libgtk2.0-0 libgtk-3-0 libgbm-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xvfb
+
+      - uses: AstrumU/shared-workflows/setup@feature/KAN-206
+        name: setup env
+        with:
+          nodeVersion: ${{inputs.nodeVersion}}
+          packageManager: ${{inputs.packageManager}}
+          cacheVersion: ${{inputs.cacheVersion}}
+          pnpmVersion: ${{inputs.pnpmVersion}}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.natToken }}
+          NODE_OPTIONS: ${{ inputs.nodeOptions }}
+
+      - uses: actions/cache@v3
+        id: build-cache
+        name: Restore build cache
+        with:
+          path: ./*
+          key: ${{ github.sha }} 
+
+      - name: Install & verify Cypress
+        run: |
+          yarn cypress install
+          yarn cypress verify
+          yarn cypress info
+          yarn cypress version
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NODE_AUTH_TOKEN }}
```

```suggestion
          NODE_AUTH_TOKEN: ${{ secrets.NODE_AUTH_TOKEN }}
```

why are we using `NODE_AUTH_TOKEN` here but not `natToken`?